package.json

```diff
@@ -28,6 +28,7 @@
     "husky": "^7.0.4",
     "lerna": "^4.0.0",
     "prettier": "^2.5.1",
+    "pretty-quick": "^3.1.3",
```

Comment:
❓ Just curious, how come we didn't we go with the standard `lint-staged` util instead? I **think** we might want to run both `eslint --fix` (import ordering etc) and `prettier`  pre-commit

Reply:
@nikosdouvlis 
`pretty-quick` was just selected for the easy integration with husky. _(After the upgrade, there is no need for a lint_staged package.json attribute)_
We can add eslint but I would consider it out of scope for now since it will release dragons.